### ISSUE TITLE
Cannot customize my character after paying for appearence change, button has appeared

### ISSUE THREAD

**Issue by Mearyl** [dalaranwow/dalaran-wow#4821]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When i paid for donation points, it went through, i spent the donationpoints on an appearence change, and the button for then appearence change has appeared, but when i was finished making my new character it just came up with a message "Could not customize character."
**Expected behaviour**: Tell us what should happen instead
It should complete my character, and change it into what i wanted it to be and i should be able to play again after that.
**Steps to reproduce the problem**:

1. Spend donation points on appearence change
2. Wait for button to appear (payment to go through)
3. Customize and press accept, and the message "Could not customize character" should appear.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![billede](https://user-images.githubusercontent.com/30321800/28424957-471a2bee-6d6f-11e7-8c00-b8e3dfe6a409.png)

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
